bazarr/radarr/sync/movies.py

```diff
@@ -45,6 +52,16 @@ def update_movie(updated_movie, send_event):
             event_stream(type='movie', action='update', payload=updated_movie['radarrId'])
 
 
+def get_movie_monitored_status(movie_id):
+    existing_movie_monitored = database.execute(
+        select(TableMovies.monitored)
+        .where(TableMovies.tmdbId == movie_id))\
+        .first()
+    if existing_movie_monitored == None:
+        return True
+    else:
+        return bool_map[existing_movie_monitored[0]]
```

Comment:
`return bool_map[existing_movie_monitored['monitored']]`

Reply:
That doesn’t work.
I’m indexing into the tuple returned by the database call.

Reply:
You're right, my bad. The result is still a tuple of a single row.